src/index.js

```diff
@@ -18,6 +19,17 @@ document.addEventListener('DOMContentLoaded', async () => {
       <div class="like-count">Likes: 0</div>
     `;
 
+    const likeButton = card.querySelector('.like-button');
+    const likeCount = card.querySelector('.like-count');
+
```

Comment:
Nice use of the querySelector to call your variables. There is not much to say besides this because you have done a wonderful work. Cheers!